<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveValuesCheckBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual control</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateTimeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date/time</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayoutRPM">
    <item>
     <widget class="QWidget" name="RPM" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="Kp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ki">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Kd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Kp_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kp</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ki_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ki</string>
   </property>
  </widget>
  <widget class="QLabel" name="Kd_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kd</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="delay_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time delay (ms)</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayoutPWM">
    <item>
     <widget class="QWidget" name="PWM" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="reference">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="reference_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reference</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartStop">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>56</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_parameters">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Controller Parameters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveValuesCheckBox_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatic control</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
